### DISCUSSION TITLE
How to embed custom text in serenity report in typescript

### DISCUSSION THREAD

**Discussion by Rupesh-Kumar-Maersk** [serenity-js/serenity-js#1672] [How to... ?]
How to embed custom text in serenity report in typescript. I am using the combination of Playwright with Serenity. 
In java we do have a option to record data like Serenity.recordReportData().withTitle("Version").andContents(version);
do we have similar option in js

**Comment by viper3400** [comment]
Hi @Rupesh-Kumar-Maersk, the easiest would be using the [Log activity](https://serenity-js.org/api/core/class/Log/).

**Reply by Rupesh-Kumar-Maersk** [reply]
@viper3400 Thank you for responding. could you please tell is there any way to display the hyperlink directly as the embedded text in the serenity report

**Reply by viper3400** [reply]
Hi @Rupesh-Kumar-Maersk , maybe I misunderstood , but which hyperlink do you refer to?
